cmd/kraft/pkg/push/push.go

```diff
@@ -122,6 +122,7 @@ func (opts *Push) Run(cmd *cobra.Command, args []string) error {
 	if pm, compatible, err := pmananger.IsCompatible(ctx, ref); err == nil && compatible {
 		packages, err := pm.Catalog(ctx,
 			packmanager.WithCache(true),
+			packmanager.WithMultiple(true),
```

Comment:
The commit introducing this (66057790) should be moved later since this method does not yet exist at this point.

Reply:
I did a top-down commit order so everything you see in the history is basically reversed let's say

I think it made a bit more sense than bottom-up for reviewing